src/styles/List.js

```diff
@@ -20,13 +20,28 @@ const Container = styled.ul`
     background-color: ${colors.lightestGrey};
   }
 
-  .participants {
-    height: 15px;
-    border-radius: 10px;
+  .avatar-stack {
     vertical-align: middle;
-    margin-right: 0;
-    margin-left: 5px;
-    margin-top: 5px;
+    display: inline;
+    margin-left: 3px;
+
+    .avatar {
+      height: ${size.tiny};
+      margin-right: -${size.micro};
+      margin-top: 10px;
```

Comment:
I think maintaining a consistent approach to feed values either in static/dynamic form should be followed. For margin-right it comes from a variable. For margin-top it is fed static.

Reply:
Agreed, this should be:

```js
margin-top: ${margin.gutter}
```

Reply:
Done :+1: